app.js

```diff
@@ -4,5 +4,14 @@ document.addEventListener("DOMContentLoaded", () => {
   const game = new Game();
 
   // Start game
-  game.start({ healthElement: ".health" });
+  game.start({
+    healthElement: "#heart_param",
+    hungerElement: "#apple_param",
+    energyElement: "#sleep_param",
+    funElement: "#ball_param",
+  });
+
+  setInterval(() => {
+    game.update();
```

Comment:
**nitpick:** Im not sure if this game update should be here, as it should be a part of Tamagotchi class. This interval would keep running even when Tamagotchi is dead.